packages/mint-components/package.json

```
@@ -0,0 +1,74 @@
+{
+  "name": "@saasquatch/mint-components",
+  "version": "1.0.0-70",
```

Should be a 1.0 main release

```suggestion
  "version": "1.0.0",
```